qutip/entropy.py

```diff
@@ -2,8 +2,8 @@
            'concurrence', 'entropy_conditional', 'entangling_power',
            'entropy_relative']
 
-from numpy import conj, e, inf, imag, inner, real, sort, sqrt
-from numpy.lib.scimath import log, log2
+from .backend import np
+from math import e
```

Comment:
This `e` is not used...

Reply:
It is used in function definition of `entropy_relative`

Reply:
Why use the one from math instead of numpy?
Is using the numpy wrapper in a function definition causing bugs?

Reply:
I did this to future proof with any error that may occur.